### PR TITLE
Add Arkansas Tech University

### PR DESCRIPTION
**Pull request by AnalogCyan**
Add Arkansas Tech University as they are stating the process of implementing 2FA.

### CODE REVIEW

File: _data/education.yml [hunk 1]
```diff
@@ -23,6 +23,16 @@ websites:
...
     twitter: antagningse
 
+  - name: Arkansas Tech University
+    url: https://www.atu.edu/
+    img: atu.png
+    tfa:
+      - sms
+      - phone
+      - software
+    exceptions: "Requires submitting a support ticket to activate. Currently only applies to email, but they are looking into implementing it account-wide."
```
Comment (kmpoppe) [inline _data/education.yml]:
```suggestion
    exception: "Requires submitting a support ticket to activate. Currently only applies to email, but they are looking into implementing it account-wide."
```

Comment (kmpoppe) [reply _data/education.yml]: The tag has changed with the API v2 commit a few dys ago.

**Review by kmpoppe — CHANGES_REQUESTED** [review]
See inline comment

File: _data/education.yml [hunk 2]
```diff
@@ -23,6 +23,16 @@ websites:
...
     twitter: antagningse
 
+  - name: Arkansas Tech University
+    url: https://www.atu.edu/
+    img: atu.png
+    tfa:
+      - sms
+      - phone
+      - software
+    exception: "Requires submitting a support ticket to activate. Currently only applies to email, but they are looking into implementing it account-wide."
```
Comment (Carlgo11) [inline _data/education.yml:33]: If 2FA isn't required account wide it's not 2FA according to our definition.

**Comment by kmpoppe** [comment]
I'm closing this PR because the site currently doesn't fullfil our requirements for an account-wide 2FA setup.

**Pull request closed without merging**